### PR TITLE
Issue #3891: Split and Organize Checkstyle inputs by Test for AvoidInlineConditionals

### PR DESCRIPTION
**Pull request by izishared**
Issue #3891 Move test resource of AvoidInlineConditionals to new location

### CODE REVIEW

File: src/test/java/com/puppycrawl/tools/checkstyle/checks/coding/AvoidInlineConditionalsCheckTest.java [hunk 1]
```diff
@@ -35,7 +35,8 @@
     @Override
     protected String getPath(String filename) throws IOException {
         return super.getPath("checks" + File.separator
-                + "coding" + File.separator + filename);
+                + "coding" + File.separator
+                + "avoidinlineconditionals" + File.separator + filename);
     }
 
     @Test
```
Comment (Vladlis) [inline src/test/java/com/puppycrawl/tools/checkstyle/checks/coding/AvoidInlineConditionalsCheckTest.java]: Please rename this test to smth meaningful, at least "testDefault"

Comment (izishared) [inline src/test/java/com/puppycrawl/tools/checkstyle/checks/coding/AvoidInlineConditionalsCheckTest.java]: @Vladlis I've renamed this test and test recource. Please review this changes.

**Comment by izishared** [comment]
@Vladlis I've renamed this test resource by agreement.

**Comment by romani** [comment]
I did not found specific issue for AvoidInlineConditionals

@izishared , are you going to resolve #3891 completely (for all Checks) ?

**Comment by izishared** [comment]
@romani there are so many checks. I can change all checks in 'com.puppycrawl.tools.checkstyle.checks.coding' package. You don't mind?

**Pull request merged**